app/src/main/java/com/techlads/composetv/details/ProductDetailsScreen.kt

```diff
@@ -0,0 +1,198 @@
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.zIndex
+import androidx.tv.material3.MaterialTheme
+import androidx.tv.material3.Text
+import com.techlads.composetv.R
+import com.techlads.composetv.widgets.BorderedFocusableItem
+
+@Composable
+fun ProductDetailsScreen() {
+    ProductDetailsContent()
+}
+
+@Composable
+fun ProductDetailsContent() {
+    Box {
+        SearchIcon(modifier = Modifier
+            .size(80.dp)
+            .align(Alignment.TopStart)
+            .padding(24.dp)
+            .zIndex(1f)
+        )
+
+        Column(
+            Modifier
+                .align(Alignment.BottomCenter)
+                .fillMaxSize()
+        ) {
+            BannerImage(modifier = Modifier
+                .fillMaxSize()
+                .weight(.4f)
+            )
+            Column(modifier = Modifier.weight(.6f)) {
+                ButtonSection()
+                DetailsSection()
+            }
+        }
+
+        ThumbnailImageCard(modifier = Modifier
+            .align(Alignment.CenterStart)
+            .padding(start = 30.dp),
+            parent = 1, child = 1)
+    }
+}
+
+
+@Composable
+fun SearchIcon(modifier: Modifier) {
+    Image(
+        modifier = modifier,
+        painter = painterResource(id = R.drawable.ic_search),
+        contentDescription = "search",
+        contentScale = ContentScale.Crop
+    )
+}
+
+@Composable
+fun BannerImage(modifier: Modifier) {
+    Image(
+        modifier = modifier
+            .fillMaxSize()
+            .height(200.dp),
+        painter = painterResource(id = R.drawable.hero_item),
+        contentDescription = "Hero item background",
+        contentScale = ContentScale.Crop
+    )
+}
+
+@Composable
+fun ButtonSection() {
+    Row(modifier = Modifier
+        .fillMaxWidth()
+        .background(color = Color(0xFF2D2D44)),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Start
+    ) {
+        Spacer(modifier = Modifier.width(280.dp))
+        Button(
+            modifier = Modifier,
+            shape = RoundedCornerShape(4.dp),
+            contentPadding = PaddingValues(vertical = 0.dp, horizontal = 16.dp),
+            onClick = {  }) {
+            Text(modifier = Modifier.padding(horizontal = 16.dp),
+                text = "Play")
+        }
+        Spacer(modifier = Modifier.size(16.dp))
+        Text(
+            color = Color.White,
```

Comment:
Please remove hardcoded text color, see the existing implementation for light text